File: Udemy/Algoritmos e Logica de Programacao/C/Estrutura Sequencial/ExerciciosEstruturaSequencial/ProblemaPagamento.c
/*Problema "pagamento" Fazer um programa para ler o nome de um(a) funcionário(a),
o valor que ele(a) recebe por hora, e a quantidade de horas trabalhadas por ele(a).
Ao final, mostrar o valor do pagamento do funcionário com uma mensagem explicativa,
conforme exemplo.*/

#include<stdio.h>
#include<stdlib.h>

int main(){

    char nome[50];
    int horaTrabalhada;
    double valorHora, calc;

    printf("Nome: ");
    scanf(" %[^\n]s",nome);
    printf("Valor por hora: ");
    scanf("%lf",&valorHora);
    printf("Horas trabalhadas: ");
    scanf("%d",&horaTrabalhada);

    calc = (double)horaTrabalhada * valorHora;

    printf("O pagamento para %s deve ser %.2lf",nome,calc);


    return 0;
}

File: Algoritmos e Logica de Programacao/Java/ExerciciosJavaEstruturaSequencial/src/application/ProblemaPagamento.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class ProblemaPagamento {

	public static void main(String[] args) {
		/*
		 * Problema "pagamento"
Fazer um programa para ler o nome de um(a) funcionário(a), o valor que ele(a) recebe por hora, e a
quantidade de horas trabalhadas por ele(a). Ao final, mostrar o valor do pagamento do funcionário com
uma mensagem explicativa, conforme exemplo.
		 * */
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		String nome;
		double valorPorHora, horas;
		
		System.out.print("Nome: ");
		nome = sc.nextLine();
		System.out.print("Valor por hora: ");
		valorPorHora = sc.nextDouble();
		System.out.print("Horas trabalhadas: ");
		horas = sc.nextDouble();
		System.out.println("O pagamento para "+nome+" deve ser "+String.format("%.2f", valorPorHora * horas));
		
		
		sc.close();

	}

}
